tests/Microsoft.Bot.Builder.TestBot.Json/luis.settings.tomlm.westus.json

```diff
@@ -1,7 +0,0 @@
-{
-    "luis": {
-        "ToDoLuis_en-us_lu": "6d4d7ad4-e351-4e6d-914f-ae423725ff83",
-        "OrderSandwichApp_en-us_lu": "5b0c7c38-cb35-4e1a-9e4d-6d66366aac67",
-        "sandwich_en-us_lu": "9ce2a124-f654-4625-ac8d-256cf0cb6474"
-    }
-}
```

Comment:
Is it right that luis settings are deleted?

Reply:
it's fine, it's just my settings.

---
In reply to: [376591199](https://github.com/microsoft/botbuilder-dotnet/pull/3350#discussion_r376591199) [](ancestors = 376591199)